doc/source/how-to/documenting.rst

```diff
@@ -522,15 +537,47 @@ artifacts as a convenient zip and to open the documentation by opening
 Next, deploy your documentation to the ``gh-pages`` branch via using the
 ``JamesIves/github-pages-deploy-action`` action:
 
-.. code-block:: yaml
+.. tabs::
+
+    .. group-tab:: Using ``tox``
+
+        .. code-block:: yaml
+        
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                github-token: ${{ secrets.GITHUB_TOKEN }}
+                branch: gh-pages
+                folder: .tox/doc_out
+                clean: true
+
+    .. group-tab:: Without Using ``tox``
+
+        .. code-block:: yaml
 
-    - name: Deploy
-      if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-      uses: JamesIves/github-pages-deploy-action@4.3.0
-      with:
-        branch: gh-pages
-        folder: doc/build/html
-        clean: true
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                github-token: ${{ secrets.GITHUB_TOKEN }}
+                branch: gh-pages
+                folder: doc/_build/html
+                clean: true
+
+Notice that for previous job steps, a ``GITHUB_TOKEN`` is required. This repo
+needs ``write`` rights and SSO access if you plan to deploy to the ``gh-pages``
+of another branch.
+
+To create the ``TOKEN``, go to the ``Settings`` section in your GitHub profile.
+In the left side bar, select the ``Developer Settings`` section and then
+``Personal access tokens``. Finally, click on ``Generate new token`` and give it
+``write`` permissions. You will be prompted with the value of the ``TOKEN``.
+Finally, click on ``Configure SSO`` to allow using it with the PyAnsys
+repositories you have access to.
```

Comment:
Marina complained that she got authentication errors even she did approve the SSO.
Hence I would add:

```suggestion
Finally, click on ``Configure SSO`` to allow using it with the PyAnsys
repositories you have access to.

.. note::
   In some cases, the authentification might need specific approval. 
   If you do still get authentification errors such as:
   
   .. code:: python
      :emphasize-lines: 1,2,9
      
      remote: The `pyansys' organization has enabled or enforced SAML SSO. To access
      remote: this repository, visit https://github.com/orgs/pyansys/sso?            authorization_request=AGWYQUM5VKPHAQHRS2H3JNTCVBNE7A5PN5ZGOYLONF5GC5DJN5XF62LEZYB663VUVVRX      EZLEMVXHI2LBNRPWSZGOGU33VEFPMNZGKZDFNZ2GSYLML52HS4DFVNHWC5LUNBAWGY3FONZQ
      remote: and try your request again.
      fatal: unable to access 'https://github.com/pyansys/pynexus-dev-docs.git/': The requested URL returned error: 403
      Running post deployment cleanup jobs… 🗑️
      /usr/bin/git checkout -B github-pages-deploy-action/x00pqaqlu
      Reset branch 'github-pages-deploy-action/x00pqaqlu'
      /usr/bin/chmod -R 777 github-pages-deploy-action-temp-deployment-folder
      /usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
      Error: The deploy step encountered an error: The process '/usr/bin/git' failed with exit code 128 ❌
      
   Please follow the link in the log to authorize the workflow.
```